python/lsst/obs/base/gen2to3/repoConverter.py

```diff
@@ -184,7 +185,8 @@ class RepoConverter(ABC):
     task : `ConvertRepoTask`
         Task instance that is using this helper object.
     root : `str`
-        Root of the Gen2 repo being converted.
+        Root of the Gen2 repo being converted.  Will be converted to an
+        absolute path if necessary.
```

Comment:
If we're using `realpath`, should this be "absolute path, resolving all symlinks if necessary"?